guides/getting_started/comparison_to_alternatives.md

```diff
@@ -0,0 +1,112 @@
+# Comparison to alternatives
+
+## About MeiliSearch
+
+MeiliSearch has been designed to be a ready-to-go open source solution and deliver an intuitive and instantaneous search experience. As the amount of information available on the websites increases as time goes by, it is essential to provide users with the most relevant results. Since many closed source search engines and open source alternatives can be used, determining which one to install is critical and may not seem easy at first.  
+
+The present article covers a comparison of alternatives to MeiliSearch, i.e. available search technologies to be considered when planning to implement a search bar.  
+
+## Comparisons
+
+### MeiliSearch vs. Elasticsearch
+
+Elasticsearch has been designed as a backend search engine and, although it is not at first suited for this purpose, is commonly used to build search bars for the end users.  
+Unlike Elasticsearch which is a general search engine, MeiliSearch focuses on delivering a specific kind of features.  
+
+Elasticsearch can handle search through massive amounts of data and perform text analysis. In order to make it effective for end-user searching, you need to spend time understanding more about how Elasticsearch works internally to be able to customize and tailor it to fit your needs.  
+MeiliSearch is intended to deliver performant instant search experiences aimed at end-users. However, processing complex queries or analyzing large datasets (over 100 M documents) is not possible.  
```

Comment:
`Not possible` or a `significantly loss of speed`?

Reply:
```suggestion
MeiliSearch is intended to deliver performant instant search experiences aimed at end-users. However, processing complex queries or analyzing very large datasets is not possible. 
```